src/main/java/com/gmail/controller/UserController.java

```diff
@@ -0,0 +1,43 @@
+package com.gmail.controller;
+
+import com.gmail.dto.User;
+import com.gmail.dto.UserDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+/**
+ * Created by Space on 17.05.2019.
+ */
+@Controller
+public class UserController {
+    @Autowired
+    private UserDto userDto;
```

Comment:
use in  controller service layer
read about MVC pattern

Reply:
and better use di with constructor
https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/